### PR TITLE
Update whats-new-in-defender-for-office-365.md

### PR DESCRIPTION
**Pull request by MishraSoumyaMS**

### CODE REVIEW

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 1]
```diff
@@ -42,7 +42,12 @@ For more information on what's new with other Microsoft Defender security produc
 - [What's new in Microsoft Defender for Identity](/defender-for-identity/whats-new)
 - [What's new in Microsoft Defender for Cloud Apps](/cloud-app-security/release-notes)
 
+## October 2022
+- [Automated Investigations email cluster action de-duplication](air-review-approve-pending-completed-actions.md) We are putting additional checks if the same investigation cluster is already approved in last 1 hour, new duplicate remediation will not be processed again.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
- [Automated Investigations email cluster action de-duplication](air-review-approve-pending-completed-actions.md): We have added additional checks. If the same investigation cluster is already approved during the past hour, new duplicate remediation will not be processed again. 
```

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 2]
```diff
@@ -42,7 +42,12 @@ For more information on what's new with other Microsoft Defender security produc
 - [What's new in Microsoft Defender for Identity](/defender-for-identity/whats-new)
 - [What's new in Microsoft Defender for Cloud Apps](/cloud-app-security/release-notes)
 
+## October 2022
+- [Automated Investigations email cluster action de-duplication](air-review-approve-pending-completed-actions.md) We are putting additional checks if the same investigation cluster is already approved in last 1 hour, new duplicate remediation will not be processed again. 
+
 ## September 2022
+**Automatic redirection from Office Action Center to Unified Action Center:** Action center in the Email & collaboration blade Email & Collaboration > Review > Action center( https://security.microsoft.com/threatincidents) is automated reditected to Actions & Submissions > Action center -> History tab (https://security.microsoft.com/action-center/history)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
**Automatic redirection from Office action center to unified action center:** The action center in the Email & collaboration blade - **Email & Collaboration** > **Review** > **Action center**(https://security.microsoft.com/threatincidents) is automatically redirected to **Actions & Submissions** > **Action center** > **History** (https://security.microsoft.com/action-center/history).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MishraSoumyaMS, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@MSFTTracyP Gentle reminder

**Comment by v-ratulach** [comment]
Created new PR with these changes: https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/20331. Closing this PR and publishing the new one.

**Pull request closed without merging**